frontend/modules/browse/components/Filter.js

```diff
@@ -32,23 +32,29 @@ export default injectIntl(React.createClass({
       }
     });
 
-    var _onClick = this._onClick;
-    var active = this.props.active;
-    var items = this.props.data.map(function(item) {
-      var classNames = "list-group-item";
-      if (item.title == active) {
-        classNames += " active";
+    const active = this.props.active;
+    const items = this.props.data.map((item) => {
+      let classNames = ["list-group-item"];
+      if (item.slug == active) {
+        classNames += [" active"];
       }
+
+      if (item.total == 0) {
+        return null;
+      }
```

Comment:
We could, but then we'd need a different endpoint for when we query the API for all cuisines/courses in the `recipe_form` (or at the very least a way to tell the endpoint to return all cuisines/courses or only ones with associated recipes). Seems like a bit of overkill at the moment.